subprojects/build-operations/src/main/java/org/gradle/trace/pid/AbstractPidCollectorBuildService.java

```diff
@@ -26,7 +28,11 @@ public interface Parameters extends BuildServiceParameters {
     }
 
     protected AbstractPidCollectorBuildService() {
-        GFileUtils.writeFile(getPid().toString(), getParameters().getPidFile().get().getAsFile());
+        try (FileWriter writer = new FileWriter(getParameters().getPidFile().get().getAsFile())) {
+            writer.write(getPid().toString());
```

Comment:
💅 Can we maybe simplify it and just use `Files.write`?:
```
        try {
            Files.write(getParameters().getPidFile().get().getAsFile().toPath(), getPid().toString().getBytes(StandardCharsets.UTF_8));
        } catch (IOException e) {
            throw new UncheckedIOException(e);
        }
 ```
 ?

Reply:
I don't have a strong opinion about it, but it feels more mouthful. I've updated it